BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -13,6 +13,9 @@ public class PerformanceConfiguration
 
         private const string LEGACY_DEFAULT_ENDPOINT = "https://otlp.bugsnag.com/v1/traces";
         private const string DEFAULT_ENDPOINT = "https://{0}.otlp.bugsnag.com/v1/traces";
+        private const string HUB_ENDPOINT = "https://{0}.insighthub.smartbear.com/v1/traces";
+        private const string HubApiPrefix = "00000";

```

Comment:
nit: maybe `HubApiPrefix` should match the formatting of these other constants in this case?
```suggestion
        private const string HUB_API_PREFIX = "00000";

```